mmdet/core/bbox/assigners/max_iou_assigner.py

```diff
@@ -88,6 +97,12 @@ def assign(self, bboxes, gt_bboxes, gt_bboxes_ignore=None, gt_labels=None):
             overlaps[:, ignore_max_overlaps > self.ignore_iof_thr] = -1
 
         assign_result = self.assign_wrt_overlaps(overlaps, gt_labels)
+        # trun the assign results back to cuda
```

Comment:
A typo here: trun -> turn.
"move" or "convert" may be better.

Reply:
ok I have made the change